sdcm/sct_events/database.py

```diff
@@ -83,7 +83,7 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
 
 
 # cause this is warning level, it's need to be before WARNING being suppressed
-DatabaseLogEvent.add_subevent_type("OVERSIZED_ALLOCATION", severity=Severity.ERROR,
+DatabaseLogEvent.add_subevent_type("OVERSIZED_ALLOCATION", severity=Severity.WARNING,
```

Comment:
I'd propose to reduce severity in `enable_default_filters` with `SkipPerIssues` referencing to issue

Reply:
If it was one specific issue, yeah.

But it's an umbrella for multiple of those, it doesn't worth the back and forth for each.

Reply:
ok, just need to remember to take it off when they're fixed

Reply:
it would take a while to chase them all in master (hopefully until 2025.3 is branched, or we'll need to repeat this one)